voyager-rxjava/build.gradle.kts

```diff
@@ -0,0 +1,60 @@
+plugins {
+    kotlin("multiplatform")
+    id("com.android.library")
+    id("com.vanniktech.maven.publish")
+}
+
+kotlin {
+    explicitApi()
+
+    android {
+        publishAllLibraryVariants()
+    }
+    jvm("desktop")
+
+    sourceSets {
+        /* Source sets structure
+        common
+          ├─ jvm
+              ├─ android
+              ├─ desktop
+         */
+        val commonMain by getting
+        val jvmMain by creating {
+            dependsOn(commonMain)
+            dependencies {
+                api(projects.voyagerCore)
+                implementation(libs.compose.rxjava)
```

Comment:
Can you replace the `implementation(libs.compose.rxjava)` with `compileOnly(libs.compose.rxjava)` to follow the fix from #35 ?